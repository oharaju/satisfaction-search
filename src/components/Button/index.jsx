import React from 'react';

export default function Button() {
  return(
    <div>
      <button class="btn-main">Teste Vercel</button>
    </div>
  )
}
