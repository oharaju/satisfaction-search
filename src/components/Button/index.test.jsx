import renderer from 'react-test-renderer';
import React from 'react';
import Button from './index';

it('renders correctly', () => {
  const tree = renderer
    .create(<Button/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});