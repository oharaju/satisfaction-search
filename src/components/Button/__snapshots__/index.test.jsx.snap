// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <button
    class="btn-main"
  >
    Teste Vercel
  </button>
</div>
`;
